march_gait_selection/src/march_gait_selection/state_machine/setpoints_gait.py

```diff
@@ -4,6 +4,12 @@
 
 
 class SetpointsGait(GaitInterface, Gait):
+    def __init__(self, gait_name, subgaits, graph):
+        super(SetpointsGait, self).__init__(gait_name, subgaits, graph)
+        self._current_subgait = None
+        self._should_stop = False
+        self._subgait_duration = 0.0
```

Comment:
```suggestion
        self._time_since_start = 0.0
```
Woospie same goes for this one of course